api/src/wfl/executor.clj

```diff
@@ -192,6 +178,15 @@
                     {:executor executor
                      :object   object}))))
 
+(defn ^:private warn-on-method-config-version-mismatch
+  "Log warning if `methodConfigVersion` does not match `expected`."
+  [{:keys [methodConfigVersion] :as methodconfig} expected]
+  (when-not (= expected methodConfigVersion)
+    (log/warning "Unexpected method configuration version"
+                 :expected            expected
+                 :actual              methodConfigVersion
+                 :methodConfiguration methodconfig)))
+
```

Comment:
`methodConfigVersion` is in `methodconfig`.
Do you just want to emphasize it in the log?

Reply:
Yes, it was originally to emphasize it in the log but I don't think that's really necessary.  Here's an actual log (with its old severity of ERROR):

https://cloudlogging.app.goo.gl/KPvujUxwSWstaZWw8